prediction/src/tests/test_load_ct.py

```diff
@@ -68,11 +68,11 @@ def test_load_meta(metaimage_path, dicom_path):
 def test_metadata(metaimage_path, dicom_path):
     meta = load_ct.load_ct(dicom_path, voxel=False)
     meta = load_ct.MetaData(meta)
-    zipped = zip(meta.spacing, (0.703125, 0.703125, 2.5))
+    zipped = zip(meta.spacing, (2.5, 0.703125, 0.703125))
     assert all([m_axis == o_axis for m_axis, o_axis in zipped])
 
     meta = load_ct.load_ct(metaimage_path, voxel=False)
-    spacing = list(reversed(meta.GetSpacing()))
+    spacing = meta.GetSpacing()[::-1]
```

Comment:
Hm? :)

Reply:
The `SimpleITK` reads CT scan array from MetaImages in `zyx` order, methods like GetSpacing | GetOrigin return values in `xyz` order, though.

Reply:
Can you add this as a comment in the code? :)